src/Samples/SubModel/Program.fs

```diff
@@ -4,6 +4,41 @@ open System
 open Elmish
 open Elmish.WPF
 
+module Counter =
+
+  type Model =
+    { Count: int
+      StepSize: int }
+  
+  let init () =
+    { Count = 0
+      StepSize = 1 }
+  
+  type Msg =
+    | Increment
+    | Decrement
+    | SetStepSize of int
+    | Reset
+    
+  let canReset = (<>) (init ())
```

Comment:
Same as previous comment: Inline or write as not point-free.

Reply:
Reverted to inline